src/shared/components/carousel/carousel.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import Slider from 'react-slick';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import * as styles from './carousel.css';
+import SlideIndicator from '../carousel/slideindicator/slideIndicator';
+
+interface CarouselProps {
+  children: React.ReactNode;
```

Comment:
```suggestion
  children: ReactNode;
```
이렇게 수정해주세요! 이유는 승택님 PR에~!

Reply:
import방식으로 수정했슴니다